### DISCUSSION TITLE
HTMLCanvasElement is not defined

### DISCUSSION THREAD

**Discussion by MangMax** [nuxt/nuxt#21874] [Questions]
I'm trying to introduce vue-cesium in nuxt and I get an HTMLCanvasElement is not defined error when starting up.
![image](https://github.com/nuxt/nuxt/assets/68525017/43472e8f-90d9-4959-98e5-5a5c96fb57da)

**Comment by wobsoriano** [comment]
Wrap it with a `<ClientOnly />` component
